en/flying/missions.md

```diff
@@ -27,6 +27,23 @@ By default this is the next waypoint.
 
 Vehicle types that cannot independently control yaw and direction of travel will ignore yaw settings (e.g. Fixed Wing).
 
+### Setting Acceptance/Turning Radius
+
+The *acceptance radius* defines the circle around a waypoint within which a vehicle considers it has reached the waypoint, and will immediately switch to (and start turning towards) the next waypoint.
+
+For a multi-rotor drones, the acceptance radius is tuned using the parameter [NAV_ACC_RAD](../advanced_config/parameter_reference.md#NAV_ACC_RAD).
+By default, the radius is small to ensure that multirotors pass above the waypoints, but it can be increased to create a smoother path such that the drone starts to turn before reaching the waypoint.
+
+The image below shows the same mission flown with different acceptance radius parameters:
+
+![acceptance-radius-comparison](../../assets/flying/acceptance_radius_comparison.png)
+
+The speed in the turn is automatically computed based on the acceptance radius (= turning radius) and the maximum allowed acceleration and jerk (see [Jerk-limited Type Trajectory for Multicopters](../config_mc/mc_jerk_limited_type_trajectory.md#auto-mode)).
+
+:::tip

```

Comment:
FYI, @bresch This is all good. All I have really done here is push add the note below that cross links to your other docs.

Reply:
Cool, thanks!